.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 repos:
   # isort should run before black as black sometimes tweaks the isort output
   - repo: https://github.com/PyCQA/isort
-    rev: 5.4.2
```

Comment:
Let's keep this aligned with the CI version?

Reply:
at the moment, the CI doesn't pin `isort`, it uses `py38.yml`